### ISSUE TITLE
IPhone can't display gauges

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#203]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code:
1. Using an IPhone 3G. Enter http:// 
code.google.com/intl/iw/apis/visualization/documentation/gallery/
gauge.html
2. The gauges display the text but not the gauge itself.
3. The same problems appears in http:// 
code.google.com/apis/ajax/playground/?type=visualization (choose Gauge)


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?
Gauge only.


Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
NO


What operating system and browser are you using?
IPhone 3.1.2 (7D11), Safari



*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `marcelo%eshbel.com@gtempaccount.com` on 2010-02-24 10:29:12

**Comment by orwant** [comment]
```
Great if it would work with Iphone
```

Original issue reported on code.google.com by `ACE4TECH` on 2010-03-20 21:16:20

**Comment by orwant** [comment]
```
I also need this for iPhone.  I had a quick play with this and notice there is no
xmlns="http://www.w3.org/2000/svg" specification on the <svg> tag.  Could be that?
 I
only tested this with a .html page loading cut and pasted svg generated code from the
js into the attached .svg file.  The attached svg renders on iPhone.
```

Original issue reported on code.google.com by `aarondevenny` on 2010-03-24 08:25:31

<hr>
- _Attachment: [intro.svg](https://storage.googleapis.com/google-code-attachments/google-visualization-api-issues/issue-203/comment-2/intro.svg)_

**Comment by orwant** [comment]
```
I have tested the comment (2) regarding te <SVG> tag.
I am not an expert so perhaps the test was no good?
See attached file
```

Original issue reported on code.google.com by `ACE4TECH` on 2010-03-24 10:03:10

<hr>
- _Attachment: [NoIphone.xhtml](https://storage.googleapis.com/google-code-attachments/google-visualization-api-issues/issue-203/comment-3/NoIphone.xhtml)_

**Comment by orwant** [comment]
```
Try Opera instead of Safari, works fine!
```

Original issue reported on code.google.com by `ACE4TECH` on 2010-04-14 11:26:09

**Comment by orwant** [comment]
```
Fails in IPAD also.
```

Original issue reported on code.google.com by `srikanthd` on 2010-04-27 23:47:50

**Comment by orwant** [comment]
```
I can replicate this issue as well and looking for a fix.
```

Original issue reported on code.google.com by `satorsdotcom` on 2010-06-06 00:26:42

**Comment by orwant** [comment]
```
Desperately in need of solution. We report dutch politic pols and want to use gauge
in one of our visualisation, from next week. As we have few thousand iPhone/iPad readers
we do need to have solution that works properly on iPhone/iPad.
```

Original issue reported on code.google.com by `dusan.mijatovic@ipsos.com` on 2011-03-11 12:29:43

**Comment by orwant** [comment]
```
Nothing is going to change with this issue, it's already more than a year not fixed.
I am using raphael.js (http://raphaeljs.com/) and janapriya's gauges (http://www.janapriya.net/vis/),
they work very well on the iphone (but not on adnroid, because they have disabled svg
on their webkit based browser :(
I have contacted the author of the janapriya gauges, they are licensed Apache2.
```

Original issue reported on code.google.com by `geg@phptalk.com` on 2011-03-11 13:17:25

**Comment by orwant** [comment]
```
Thanks for the notification. I should better look at the year when issue was rised.
I had quick look at the gauges on janapriya. 
Are there any js examples how to create an circular gauge and which libraries are used?
```

Original issue reported on code.google.com by `dusan.mijatovic@ipsos.com` on 2011-03-11 13:49:10

**Comment by orwant** [comment]
```
No, but look at the page source, the lib is easy to understand and to extend
```

Original issue reported on code.google.com by `geg@phptalk.com` on 2011-03-11 15:50:01

**Comment by orwant** [comment]
```
First time 'caller'.  Just wondering whether there is anybody from Google looking at
issues like this at all?
```

Original issue reported on code.google.com by `andaweb` on 2011-03-18 21:38:10

**Comment by orwant** [comment]
```
Ditto, be great to see these working on iPhone.
```

Original issue reported on code.google.com by `info@thewebvideo.co` on 2011-05-31 14:32:09

**Comment by orwant** [comment]
```
fixed ?? not yet ?? will it ever be fixed ?
```

Original issue reported on code.google.com by `louatiamin` on 2011-07-13 15:09:10

**Comment by orwant** [comment]
```
The issue is that many of the visualizations use Flash, so they aren't going to work
on iOS devices. Google just needs to move them to something other than Flash that's
actually a web standard. =/
```

Original issue reported on code.google.com by `rcstasel` on 2011-07-13 18:31:19

**Comment by orwant** [comment]
```
There's a working lib that works on iPads here  http://code.google.com/p/jsgauge/

It's a little quirky, but for the most basic needs will work.

My question is why isn't the GVIZ team looking at this code for use on iPad and similar
devices?
```

Original issue reported on code.google.com by `nabeel.azar` on 2011-07-25 13:53:48

**Comment by orwant** [comment]
```
This issue should be solved now, can you please re-check?
```

Original issue reported on code.google.com by `yinnon@google.com` on 2011-08-03 19:32:40

**Comment by orwant** [comment]
```
My IPhone version is 4.2.1 (8C148) now.
The Safari version on it is 5.0.2.
The gauges are still invisible (except for the labels attached to it).
I rechecked the issue on http://code.google.com/apis/visualization/documentation/gallery.html
and also on
http://code.google.com/apis/chart/interactive/docs/gallery/gauge.html



```

Original issue reported on code.google.com by `marcelo%eshbel.com@gtempaccount.com` on 2011-08-04 05:47:48

**Comment by orwant** [comment]
```
Same issue here in safari.  I only see labels. It does work fine in opera mini now though.
  4.3.2 iOS. 
```

Original issue reported on code.google.com by `tonyroomz` on 2011-08-04 08:19:06

**Comment by orwant** [comment]
```
We've jumped the gun a bit. This is fixed only in the 1.1 version for now. Hopefully
it'll hit 1.0 soon.
```

Original issue reported on code.google.com by `jinji.viz` on 2011-08-04 13:49:58

**Comment by orwant** [comment]
```
The fix is now (finally) live in 1.0.
```

Original issue reported on code.google.com by `jinji.viz` on 2011-08-21 11:06:52

**Comment by orwant** [comment]
```
It works!
Thank you.
```

Original issue reported on code.google.com by `marcelo%eshbel.com@gtempaccount.com` on 2011-08-21 12:12:38
